### DISCUSSION TITLE
✅ 🔗 Popup with dynamic content (Duplicate of #7106)

### DISCUSSION THREAD

**Discussion by Brett-0** [elementor/elementor#25473] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

You can not load dynamic content into a Popup

### Describe the solution you'd like.

Being able to have a popup or model with dynamic content

### Describe alternatives you've considered.

There are some plugins that do this but it should be a core function of Elementor. Another alternative would be to move to Blocks.

### Additional context

This is such a glaring problem. I'm an experienced developer, it's really baffling and infuriating that there have been so many people asking for this for years. It is a request many of my clients want their sites to be able to do. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by felmerald** [comment]
@Brett-0 I created the same request weeks ago. This is the request https://github.com/orgs/elementor/discussions/25377
you can also vote on my request

**Comment by arekuki** [comment]
+1

**Comment by mkarim1378** [comment]
so how could I create pop up with dynamic content in arsanet[dot]org/wp-plugins/* ?!

**Comment by mcrowell** [comment]
I would love to have added to Elementor!
Anytime I need to use Javascript or dynamic content in a popup, I have to use JetPopup.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #7106

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.26.0! 🥳 

You can now use the **Off-Canvas** widget inside your **Loop Item** templates to display additional Dynamic Content for each post in your **Loop Grid** widgets! 🎉 🥳 🙌 

You can learn more about the Off-Canvas widget (and how to use it) here: 
https://elementor.com/help/off-canvas-widget/
And about the Loop Grid here: 
https://elementor.com/help/loop-grid/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
